node/src/components/fetcher/metrics.rs

```diff
@@ -0,0 +1,57 @@
+use prometheus::{IntCounter, Registry};
+
+#[derive(Debug)]
+pub struct FetcherMetrics {
+    /// Number of fetch requests that found an item in the storage.
+    pub(super) found_in_storage: IntCounter,
+    /// Number of fetch requests that fetcher an item from peer.
```

Comment:
```suggestion
    /// Number of fetch requests that fetched an item from peer.
```

Reply:
Done in [`9960350` (#925)](https://github.com/CasperLabs/casper-node/pull/925/commits/99603501fb81193ac6820dc9eac965ab73cb98b8).